utils/proxy/rc_mocked_responses_asm_activate_only.json

```
@@ -0,0 +1,15 @@
+[
+  {
+    "client_configs" : [
+      "datadog/2/ASM_FEATURES/asm_features_activation/config"
+    ],
+    "roots" : [],
+    "target_files" : [
+      {
+        "path" : "datadog/2/ASM_FEATURES/asm_features_activation/config",
+        "raw" : "eyJhc20iOnsiZW5hYmxlZCI6dHJ1ZX19"
+      }
+    ],
+    "targets" : "eyJzaWduZWQiOnsiZXhwaXJlcyI6IjIwMjItMDktMTdUMTI6NDk6MTVaIiwic3BlY192ZXJzaW9uIjoiMS4wLjAiLCJ0YXJnZXRzIjp7ImRhdGFkb2cvMi9BU01fRkVBVFVSRVMvYXNtX2ZlYXR1cmVzX2FjdGl2YXRpb24vY29uZmlnIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjE1OTY1OGFiODViZTcyMDc3NjFhNDExMTE3MmIwMTU1ODM5NGJmYzc0YTFmZTFkMzE0ZjIwMjNmN2M2NTZkYiJ9LCJsZW5ndGgiOjI0fX0sInZlcnNpb24iOjIzMzM3MzkzfSwic2lnbmF0dXJlcyI6W3sia2V5aWQiOiJURVNUX0tFWV9JRCIsInNpZyI6IjUwNzM1YTg5MmI5ZTViOTdhZjljMGRmMTJjY2Q5MTczNDBkYjQzNzBmYWJkNWY1M2ViNzBmNjZmM2NlNDg0NDVkMjFkMGYwZWFkZmJlZWY4NGI4MDhmYjI3NjUxYzU4NWVjZWQ4MDZkOTUzZDlmNDQzNmE2ZDQ4ZjM1MGE3ZjBjIn1dfQ=="
```

`targets.signed.custom.opaque_backend_state` field is missing. Nowhere was written that this field was optional so it makes the nodejs lib consider the payload as invalid and skip the update.
```suggestion
    "targets" : "eyJzaWduZWQiOnsiZXhwaXJlcyI6IjIwMjItMDktMTdUMTI6NDk6MTVaIiwic3BlY192ZXJzaW9uIjoiMS4wLjAiLCJjdXN0b20iOnsib3BhcXVlX2JhY2tlbmRfc3RhdGUiOiJhYWFhYSJ9LCJ0YXJnZXRzIjp7ImRhdGFkb2cvMi9BU01fRkVBVFVSRVMvYXNtX2ZlYXR1cmVzX2FjdGl2YXRpb24vY29uZmlnIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjE1OTY1OGFiODViZTcyMDc3NjFhNDExMTE3MmIwMTU1ODM5NGJmYzc0YTFmZTFkMzE0ZjIwMjNmN2M2NTZkYiJ9LCJsZW5ndGgiOjI0fX0sInZlcnNpb24iOjIzMzM3MzkzfSwic2lnbmF0dXJlcyI6W3sia2V5aWQiOiJURVNUX0tFWV9JRCIsInNpZyI6IjUwNzM1YTg5MmI5ZTViOTdhZjljMGRmMTJjY2Q5MTczNDBkYjQzNzBmYWJkNWY1M2ViNzBmNjZmM2NlNDg0NDVkMjFkMGYwZWFkZmJlZWY4NGI4MDhmYjI3NjUxYzU4NWVjZWQ4MDZkOTUzZDlmNDQzNmE2ZDQ4ZjM1MGE3ZjBjIn1dfQ=="
```